pyproject.toml

```diff
@@ -33,6 +33,7 @@ typing_extensions = "^3.7.4"
 opentelemetry-api = {version = "^0.13b0",optional = true}
 opentelemetry-sdk = {version = "^0.13b0",optional = true}
 python-dateutil = "^2.7.0"
+cached-property = "^1.5.2"
```

Comment:
is there any reason to use this library instead of `functools.cached_property`?

Reply:
I'd love to use that, but we still support python 3.7. the library is basically a backport.

Reply:
Oh, I missed that. Thanks